components/src/Form/Form.story.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Field from "../Field/Field";
+import Input from "../Input/Input";
+import Form from "./Form";
+import FormSection from "./FormSection";
+
+storiesOf("Form", module)
+  .add("Form", () => (
+    <Form
+      title="New Profile"
```

Comment:
Can you try putting the attribute on the same line?
I'm trying to see which one's more readable.